2.Nomic_nELISA_Analysis/0.Exploratory_Analysis/PBMC/2.cytokine_visualization.ipynb

```diff
@@ -0,0 +1,209 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # plot scatter plot of all the treatment groups for IL-6
</pre><p>Recommend removing this code comment since in this code block you are making the scatter plot for IL-1 Beta not IL-6.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>